diff --git a/library/include/borealis/core/logger.hpp b/library/include/borealis/core/logger.hpp
index 7efb49a1..9dacd376 100644
--- a/library/include/borealis/core/logger.hpp
+++ b/library/include/borealis/core/logger.hpp
@@ -99,7 +99,7 @@ class Logger
         if (sceRtcGetCurrentClockLocalTime)
             sceRtcGetCurrentClockLocalTime(&lt);
 #else
-        std::tm time_tm = fmt::localtime(std::chrono::system_clock::to_time_t(now));
+        std::tm time_tm = std::localtime(&std::chrono::system_clock::to_time_t(now));
 #endif
         std::string log = fmt::format(format, std::forward<Args>(args)...);
 
